From 305535e303032814b65bf6d889a95f00f08a9071 Mon Sep 17 00:00:00 2001
From: Nils Philippsen <nils@redhat.com>
Date: Wed, 5 May 2010 12:49:02 +0200
Subject: [PATCH] patch: epson-expression800

Squashed commit of the following:

commit 3b501d7499357438a1fbd63fefb2f977ae3051f5
Author: Nils Philippsen <nils@redhat.com>
Date:   Wed May 5 12:14:23 2010 +0200

    Improve Epson Expression 800

    Epson Expression 800 models announce themselves as "processor", not
    "scanner".
---
 doc/descriptions/epson.desc  |    1 +
 doc/descriptions/epson2.desc |    1 +
 2 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/doc/descriptions/epson.desc b/doc/descriptions/epson.desc
index a22325c..55a0136 100644
--- a/doc/descriptions/epson.desc
+++ b/doc/descriptions/epson.desc
@@ -174,6 +174,7 @@
 :model "Expression 800"
 :interface "SCSI"
 :status :complete
+:scsi "EPSON" "Expression800" "processor"
 
 :model "Expression 1600"
 :interface "SCSI USB IEEE-1394"
diff --git a/doc/descriptions/epson2.desc b/doc/descriptions/epson2.desc
index 9a14f4f..56cabcd 100644
--- a/doc/descriptions/epson2.desc
+++ b/doc/descriptions/epson2.desc
@@ -241,6 +241,7 @@
 :model      "Expression 800"		; command spec
 :interface  "SCSI"
 :status     :complete
+:scsi       "EPSON" "Expression800" "processor"
 :comment    "overseas version of the GT-9600"
 
 :model      "Expression 836XL"		; command spec
-- 
1.6.6.1

